lambda/src/main/scala/com/gu/meeting/ReminderHandlerSteps.scala

```diff
@@ -68,41 +52,66 @@ object ReminderHandlerSteps extends StrictLogging {
     }
 
   }
+
 }
 
 case class ChatMessage(
-  text: String,
-  formattedText: String
+    text: String,
+    formattedText: String,
 ) derives Encoder
 
 case class MeetingData(
-  webHookUrl: String,
-  start: OffsetDateTime,
-  meetLink: Option[String],
-  title: String,
+    webHookUrl: String,
+    start: OffsetDateTime,
+    meetLink: Option[String],
+    title: String,
 )
 
-object MeetingData {
-  def fromApiEvent(event: Event, lambdaStartTime: OffsetDateTime): Option[MeetingData] = {
+object MeetingData extends StrictLogging {
+
+  def eventsToMessages(minuteOfInterest: OffsetDateTime, eventsList: List[Event]): List[(String, ChatMessage)] = {
+    val chatMessages =
+      for {
+        event <- eventsList
+        meeting <- MeetingData.fromApiEvent(event)
+        if minuteOfInterest == meeting.start
+        _ = logger.info("Sending message for meeting " + meeting)
+        chatMessage = {
+          val link = meeting.meetLink match {
+            case Some(value) => s"<$value|${meeting.title}>"
+            case None => meeting.title
+          }
+          val suffix = s" is starting at " + meeting.start.format(DateTimeFormatter.ofPattern("hh:mm a", Locale.UK))
```

Comment:
if you leave Locale unset, it uses US in the build and UK locally, which causes "a" to produce PM in US and pm in UK locale, thus failing the tests.